src/pages/signup/index.page.tsx

```diff
@@ -0,0 +1,34 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import styles from './Signup.module.scss';
+import SignupForm from '@/pages/signup/components/SignupForm';
+
+function Signup() {
+  return (
+    <div className={styles.container}>
+      <header className={styles.header}>
+        <Link href="/">
```

Comment:
```suggestion
        <Link href="/">
```
여기도 url 상수화 사용해도 좋을 것 같네요!